CHANGELOG.md

```diff
@@ -7,6 +7,34 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [Unreleased]
 
 ## [11.7.0]
+### Added
+- Added auto-suggestion for ticker symbols in the network form ([#21843](https://github.com/MetaMask/metamask-extension/pull/21843))
+- Added a header to CoinGecko requests to prevent API abuse ([#22151](https://github.com/MetaMask/metamask-extension/pull/22151))
```

Comment:
We should describe this in terms that are meaningful to the metamask end user. So I would write this as "Reduce occurence of bug that can cause fiat balances of tokens to be hidden or excessively stale"

And then move it to the fixed section

Reply:
Corrected, thanks.